CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Bug fixes:
 - fix(compute): clarify fastly.toml error message when file not found ([#1556](https://github.com/fastly/cli/pull/1556))
+- fix (tls-custom): correct 'tls-custom activation enable' command parameters to reflect expected input from API ([#1562](https://github.com/fastly/cli/pull/1562))
```

Comment:
Please remove the space between `fix` and `(tls-custom)` and move it to the proper section. Thanks